apps/src/netsim/NetSimMetronome.js

```diff
@@ -77,25 +76,19 @@ NetSimMetronome.prototype.tick = function (clock) {
   if (this.pulseIntervalMillis_ === Infinity) {
```

Comment:
Infinity isn't so much a magical value to indicate state as a valid edge case, right?  If I set up infinite time between pulses, the metronome behaves in a consistent way.

I suppose this edge case (infinite pulse interval) isn't especially necessary - if omitted we'd get `pulseAge / Infinity` which is zero in JavaScript unless the pulseAge is infinite (unlikely).  Still, it seemed better to be explicit about the behavior in this case.